third-party/cmake/BuildLuarocks.cmake

```diff
@@ -37,29 +91,29 @@ endif()
 
 # Each target depends on the previous module, this serializes all calls to
 # luarocks since it is unhappy to be called in parallel.
-add_custom_command(OUTPUT ${DEPS_LIB_DIR}/luarocks/rocks/lua-messagepack
+add_custom_command(OUTPUT ${HOSTDEPS_LIB_DIR}/luarocks/rocks/lua-messagepack
```

Comment:
I know I've asked this before, but how is it that you can run tests in the cross-compiled situation?  Is in running under Wine?